cl/opinion_page/templates/view_opinion.html

```diff
@@ -90,12 +90,16 @@ <h3 class="inline"><span>Your Notes</span></h3>
             <h3><span>Summaries ({{ summaries_count|intcomma }})</span></h3>
             <p class="bottom">Judge-written summaries of this case:</p>
             <ul>
-              {% for summary in top_summaries %}
+              {% for group in top_parenthetical_groups %}
+                {% with summary=group.representative %}
                 <li>
                   {{ summary.text|truncatewords:50|capfirst }}
-                  (from <a href="{{ summary.describing_opinion.cluster.get_absolute_url }}?{{ request.META.QUERY_STRING }}">
-                    {{ summary.describing_opinion.cluster.caption|safe|truncatewords:10|v_wrapper }}</a>)
+                    (from <a href="{{ summary.describing_opinion.cluster.get_absolute_url }}?{{ request.META.QUERY_STRING }}">
+                      {# The below line needs to be all one line or spaces gets inserted where they shouldn't be #}
+                      {{ summary.describing_opinion.cluster.caption|safe|truncatewords:10|v_wrapper }}</a>{% if group.size > 1 %}
+                        <span>and {{ group.size }} similar summar{{ group.size|pluralize:"y,ies" }}</span>{% endif %})
```

Comment:
```suggestion
                        <span>and {{ group.size|add:"-1"|intcomma }} other{{ group.size|add:"-1"|pluralize }}</span>{% endif %})
```

Reply:
My only pause with something like this is it may imply that there are 12 other summaries that say *exactly that* rather than merely similar to that and I don't want to mislead. What do you think?

Reply:
I think it should be OK. Our users will figure it out because the summary page makes it more clear.